submissions/blueprint-rant/blueprint-rant.md

```diff
@@ -0,0 +1,51 @@
+## Blueprints suck <author>Conor_</author>
+
+***Mostly***, blueprints suck. First added in version `0.9.0`, blueprints allow us to create ghost copies of all sorts of machinery setups and place them anywhere in the rest of our sprawling factories. This 'copy and paste' functionality I have no issues with and I particularly enjoyed the direct copy and paste buttons and shortcuts added with the hotbar overhaul in `0.17.10`. What I cannot endorse, is the blueprint library and import string features. I think *Rseding91* said it best in his recent*ish* [AMA on reddit](https://www.reddit.com/r/factorio/comments/in5d3i/developer_technicaloriented_ama/g45ay4e?utm_source=share&utm_medium=web2x&context=3).
+
+> I would never have added the blueprint library :) Not a popular opinion but I've had it ruin a few multiplayer games I was enjoying; someone comes through with a 'base in a blueprint' and all the creativity and fun is now over; it's just 'build that'.
+- Rseding91
+
+### The goal of Factorio
+
+The goal of Factorio is, *spoiler alert*, to build a factory, but it's so much more than that. Its about optimizing and improving your production setups, whether it be to produce more of the final product, make the bare minimum required in the tiniest space possible or keep that UPS up at 60 while allow you to keep building onwards. Each and every time you build, a green circuit maker for example, you get to use your experience to make it better than the last. Maybe this time you'll use productivity modules? Maybe you wan't to try centralized green circuit production this time round, but only have basic tiers of machines. This is the challenge of Factorio, designing within the constraints of the situation and making the best of it.
+
+![](Green circuit maker screenshot)
+*A simple but large footprint green circuit maker I recently designed to allow the production of 2 belts of green circuits, using only low tier assemblers. Notice the unique and slightly odd design, because why not?.*
+
```

Comment:
Maybe here you could add a quick paragraph explaining _why_ people then tend to use blueprints. Mentioning things like:

- Looks too complicated, and not sure if you will be able to figure it out.
- Want to get to the next technology, and don't want to spend time designing another build
- The blueprint used by Nilaus looks so cool.
- My design might not be efficient enough.

Some blueprints might seem complex, but Factorio is designed to be able to break it into components. Everything from nuclear builds to 1kspm factories. And every component can be simplistic or complicated, and you only learn how to design the complicated bases once you've designed a simple one. And that journey is why I keep coming back to Factorio.

Reply:
I couldn't manage to work this into the text anywhere unfortunately. Every time I tried to include it, I couldn't quite get it to flow right and fit the context